addons/account/models/account_move.py

```diff
@@ -5544,6 +5628,13 @@ def js_assign_outstanding_line(self, line_id):
         '''
         self.ensure_one()
         lines = self.env['account.move.line'].browse(line_id)
+        if lines[0].account_id.account_type == "asset_cash":
+            st_line = self.env['account.bank.statement.line'].search([
+                ('invoice_line_ids', 'in', lines[0].id)
+            ], limit=1)
+            wizard = self.env['bank.rec.widget'].with_context(default_st_line_id=st_line.id).new({})
```

Comment:
Doesn't exist in `account` (and I think it will conflict with the ongoing refactoring of the widget :boom:)